### PR TITLE
Feature/nuc 554 zhunor transfer link button

### PR DESCRIPTION
**Pull request by zhunor**
image

### CODE REVIEW

File: src/components/LinkButton.vue [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  name: 'LinkButton',
+  props: {
+    text: {
+      type: String,
+      required: true,
+    },
+    externalUrl: {
+      type: String,
+    },
+    visibility: {
```
[the remark this answers is not in the document]
Comment (zhunor) [reply src/components/LinkButton.vue]: Visibility is removed

File: src/components/LinkButton.native.vue [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  props: {
+    text: {
+      type: String,
+      required: true,
+    },
+    externalUrl: {
+      type: String,
+      default: '',
+    },
+    visibility: {
```
[the remark this answers is not in the document]
Comment (zhunor) [reply src/components/LinkButton.native.vue]: Visibility is removed

File: src/components/VxpLink.native.vue [hunk 3]
```diff
@@ -0,0 +1,52 @@
+<template>
+  <Label v-if="externalUrl" class="vxp-link-button typ-body" :text="text" @tap="onTap" />
+  <Label v-else :text="text" class="vxp-link-button typ-body" @tap="$emit('tap', $event)" />
+</template>
+
+<script>
+import { openUrl } from 'tns-core-modules/utils/utils'; // eslint-disable-line
```
Comment (ahmetceylan) [inline src/components/VxpLink.native.vue]: please move tns imports to another file like utils.js, you can check nuweb-ui for sample implementation.

Comment (zhunor) [reply src/components/VxpLink.native.vue]: These imports are moved under utils.js & utils.native.js

**Pull request merged**